container/container_unix.go

```diff
@@ -34,6 +33,11 @@ import (
 // DefaultSHMSize is the default size (64MB) of the SHM which will be mounted in the container
 const DefaultSHMSize int64 = 67108864
 
+var (
+	errInvalidEndpoint = fmt.Errorf("invalid endpoint while building port map info")
```

Comment:
I removed all `errors.New` to not have package name collisions with `github.com/docker/docker/errors` :grimacing: